src/PROPOSAL/detail/PROPOSAL/crosssection/parametrization/PhotoMuPairProduction.cxx

```diff
@@ -0,0 +1,71 @@
+
+#include <cmath>
+
+#include "PROPOSAL/crosssection/parametrization/PhotoMuPairProduction.h"
+#include "PROPOSAL/Constants.h"
+#include "PROPOSAL/medium/Components.h"
+#include "PROPOSAL/particle/Particle.h"
+
+using namespace PROPOSAL;
+using std::make_tuple;
+
+double crosssection::PhotoMuPairProduction::GetLowerEnergyLim(
+        const ParticleDef&) const noexcept {
+    return 2. * MMU * SQRTE;
+}
+
+crosssection::KinematicLimits crosssection::PhotoMuPairProduction::GetKinematicLimits(
+        const ParticleDef&, const Component&, double energy) const noexcept {
+    // x is the integration variable here
+    KinematicLimits lim{};
+
+    if (energy <= 2. * MMU * SQRTE) {
+        lim.v_min = 0.5;
+        lim.v_max = 0.5;
+    } else {
+        lim.v_min = 0.5 * (1. - std::sqrt(1. - 2 * SQRTE * MMU / energy));
+        lim.v_max = 0.5 * (1. + std::sqrt(1. - 2 * SQRTE * MMU / energy));
+    }
+    return lim;
+}
+
+std::unique_ptr<crosssection::Parametrization<Component>>
+crosssection::PhotoMuPairSandrock::clone() const
+{
+    using param_t = std::remove_cv_t<std::remove_pointer_t<decltype(this)>>;
+    return std::make_unique<param_t>(*this);
+}
+
+double crosssection::PhotoMuPairSandrock::DifferentialCrossSection(
+        const ParticleDef&, const Component& comp, double energy, double x) const
+{
+    double aux1, aux2;
+
+    auto delta = MMU * MMU / (2. * energy * x * (1. - x));
+    auto B = 183.;
```

Comment:
The radiation logarithm depends on the medium and should be treated like in the original bremsstrahlung cross section, this parametrization derives from.
The same holds for the inelastic radiation logarithm Bprime.

Reply:
You mean
```suggestion
    auto B = comp.GetLogConstant();
```
and for Bprime we use `comp.GetBPrime()`?

Reply:
Yes, that would be more exact. 183 is the value for a Thomas-Fermi atom (approximating the electrons as an ideal Fermi gas; roughly OK for high Z), while the values from comp.GetLogConstant() are determined from a calculation using the Hartree-Fock model (self-consistent iterative determination of atomic electron wavefunctions). The change is not large, for nitrogen B is 176.6 in the Hartree-Fock method, resulting in a change of about 0.2% in the complete screening regime compared to the Thomas-Fermi value; but this way it becomes consistent to the KKP muon bremsstrahlung cross section.